<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleHW</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Bahnschrift SemiLight</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stopwatch_tap">
       <attribute name="title">
        <string>Stopwatch</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="4" column="0">
         <widget class="QPushButton" name="stopwatch_lap_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Lap</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="3">
         <widget class="QLabel" name="space">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="stopwatch_reset_btn">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QPushButton" name="stopwatch_start_stop_btn">
          <property name="text">
           <string>Start / Stop</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0" colspan="3">
         <widget class="QTextBrowser" name="stopwatch_textBrowser">
          <property name="minimumSize">
           <size>
            <width>342</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>18</pointsize>
            <underline>false</underline>
           </font>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="3">
         <widget class="QLabel" name="space_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QLabel" name="stopwatch_label">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>00:00.00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="countdown_tap">
       <attribute name="title">
        <string>Countdown Timer</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="space_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSpinBox" name="countdown_hours_spinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="hours">
            <property name="text">
             <string>Hours</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="countdown_mins_spinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="mins">
            <property name="text">
             <string>Mins</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="countdown_secs_spinBox"/>
          </item>
          <item>
           <widget class="QLabel" name="secs">
            <property name="text">
             <string>Secs</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="space_5">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="countdown_label">
          <property name="font">
           <font>
            <family>Trebuchet MS</family>
            <pointsize>40</pointsize>
           </font>
          </property>
          <property name="text">
           <string>00:00:00</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="space_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="countdown_reset_btn">
            <property name="text">
             <string>Reset</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="countdown_start_btn">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="countdown_pause_btn">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>Pause</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="time_up_label">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>19</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(221, 0, 0);</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="countdown_close_btn">
          <property name="text">
           <string>Close</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>328</width>
            <height>341</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
